### PR TITLE
[W22] Ray Lin 5/30-6/03

### PR DESCRIPTION
**Pull request by raylin**

### CODE REVIEW

File: RayLin.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   - Bug 1086911 - Update style of about:firefox
-    - ask visual for feedback
-  - Bug 1091241 - Make add-on detail page look more like a detail page
-    - FIXED
+    - feedback-
+    - issues: blurry logo and distorted icon
   - Bug 1267935 - Video controls are not correctly scaled for several video types
-    - uplifted to Aurora
-    - video control is scaled fine in latest Aurora build(05/29) now
+    - uplift to mozilla-beta? next week
```
Comment (timdream) [inline RayLin.md]: Next week? What's the target release? Aurora 48 will be Beta 48 next week...

File: RayLin.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+  - Bug 1276686 - Selection handles are displayed when tapping on the reader view toolbar
+    - checkin-needed
 - [Firefox]
   - Bug 887934 - [webvtt] Update \<video\> controls to include options for closed captioning
-    - r? UI issues are fixed in the main
-    - next week:
-      - ask visual for new CC button icon, since transparency PNG could not be brighter even opacity is set to 1
-      - rewrite Part 2 to align with WebVTT spec
+    - r+
+    - will update patch and ask for checkin once got new icon from :shorlander
```
Comment (timdream) [inline RayLin.md]: I see `checkin-needed` already set. Are we still waiting for the icon?

File: RayLin.md (hunk 1 above)
Comment (raylin) [inline RayLin.md]: yep...so no uplift is needed, thanks.

File: RayLin.md (hunk 2 above)
Comment (raylin) [inline RayLin.md]: :jaws gave another r+ this morning, I guess he hinted that we should checkin

Comment (timdream) [inline RayLin.md]: OK, let's file a dependent on "designing the new icon" and needinfo shorlander there instead?

File: RayLin.md (hunk 1 above)
Comment (timdream) [inline RayLin.md]: That means we are not fixing this in Beta 47 and Release 47 alright? It is ok?

Comment (raylin) [inline RayLin.md]: I think it's fine, hence our patch landed on Aurora(48). Uplifting to 47 is an extra for the flag should had been removed in the earlier this year.

File: RayLin.md (hunk 2 above)
Comment (raylin) [inline RayLin.md]: Sure! Though :shorlander is on https://bugzilla.mozilla.org/show_bug.cgi?id=1271765, I don't think it could be done in the near future.

**Pull request merged**